src/core/blueprint/blueprint.rs

```diff
@@ -154,6 +154,7 @@ pub struct SchemaDefinition {
 pub struct InputFieldDefinition {
     pub name: String,
     pub of_type: Type,
+    pub resolver: Option<IR>,
```

Comment:
Inputs don't need to have resolvers. Even in the implementation in this PR, you are piping the IR to the resolvers of the fields in the output type.

Reply:
Removed, thanks for the insight.